server/models/center.js

```diff
@@ -0,0 +1,72 @@
+export default (sequelize, DataTypes) => {
+  const Center = sequelize.define('Center', {
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        notEmpty: {
+          args: true,
+          msg: 'Name field cannot be empty'
+        },
+        isAlphanumeric: {
+          args: true,
+          msg: 'Allows only alphanumeric characters'
+        }
+      },
+      set(value) {
```

Comment:
what is happening here please?

Reply:
Thanks for pointing it out, I will remove this constrain